backendRoot/src/main/java/com/virtuslab/gitmachete/backend/root/GitMacheteRepositoryFactory.java

```diff
@@ -43,8 +44,9 @@
 import com.virtuslab.gitmachete.backend.impl.GitMacheteRootBranch;
 import com.virtuslab.gitmachete.backend.impl.SyncToRemoteStatus;
 
-@Slf4j(topic = "backendRoot")
 public class GitMacheteRepositoryFactory implements IGitMacheteRepositoryFactory {
+  private static final LambdaLogger LOG = LambdaLoggerFactory.getLogger("backendRoot");
```

Comment:
Use Lombok's `@CustomLog` if easily doable

Reply:
It seems to be not so easy. I had classpath related problems  :/